applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -611,6 +672,13 @@ The application supports the following configurations:
      - Release version of the application.
 
        Enables only the necessary application functionalities to optimize its performance.
+   * - Matter bridge over Wi-Fi with nRF5340 DK and nRF7002 EK
+     - :file:`---`
```

Comment:
```suggestion
     - ---
```

Reply:
@kg-nordicsemi this comment is sort of obsolete, because this PR was not rebased, but anyway Wille is right and you can fix this as a follow-up in this PR.